### DISCUSSION TITLE
Did not get copilot for free after appoved by the Github Education

### DISCUSSION THREAD

**Discussion by ericL1u** [community/community#129224] [GitHub Education]
### Hi everyone,

hi, I have renew and approved with the GitHub student account but did not get free copilot

**Comment by 69Youssef** [comment]
wait few hours and try again

**Reply by 69Youssef** [reply]
or you can try go to settings  then copilot you will see this message "you are eligible for free usage for copilot "

**Reply by Stormlocc** [reply]
> or you can try go to settings then copilot you will see this message "you are eligible for free usage for copilot "

Can you shared the link pls

**Reply by 69Youssef** [reply]
https://github.com/settings/billing/summary

**Reply by 69Youssef** [reply]
go to bottom of page
